templates/registry/markdown/notes.j2

```diff
@@ -3,7 +3,8 @@
 {%- macro add_with_limit(note) %}{% if note.note | length > 50 %}{% set ns.notes = [ns.notes, [note]] | flatten %} [{{ ns.notes | length + ns.index }}]{% elif note.note %} {{ note.note | trim }}{% endif %}{% endmacro %}
 {% macro render() %}{% if ns.notes | length > 0 %}
 {%- for note in ns.notes %}
-{% if note.name %}**[{{ns.index+loop.index}}] `{{note.name}}`:** {{ note.note | trim }}{% else -%}**[{{ns.index+loop.index}}]:** {{ note.note | trim }} {%- endif -%}
+{% if note.name %}**[{{ns.index+loop.index}}] `{{note.name}}`:**{% else -%}**[{{ns.index+loop.index}}]:**{%- endif -%}
+{% if note.note.startswith('> [!') %}{{"\n\n"}}{% else %} {% endif %}{{ note.note | trim }}
```

Comment:
can you add a brief comment here explaining? jinja already makes no sense but this is next level 😅 

> '> [!') %}{{"\n\n"}}